crates/loro-internal/src/encoding/fast_snapshot.rs

```diff
@@ -193,6 +194,53 @@ pub(crate) fn encode_snapshot<W: std::io::Write>(doc: &LoroDoc, w: &mut W) {
     );
 }
 
+pub(crate) fn encode_snapshot_at<W: std::io::Write>(
+    doc: &LoroDoc,
+    frontiers: &Frontiers,
+    w: &mut W,
+) -> Result<(), LoroEncodeError> {
+    let version_before_start = doc.oplog_frontiers();
+    doc.checkout_without_emitting(frontiers).unwrap();
+    {
+        let mut state = doc.app_state().try_lock().unwrap();
+        let oplog = doc.oplog().try_lock().unwrap();
+        let is_gc = state.store.gc_store().is_some();
+        if is_gc {
+            unimplemented!()
```

Comment:
Apart from version issues, is there anything else that needs special handling in GC mode?

Reply:
No